app/jobs/regular/export_csv_file.rb

```diff
@@ -227,7 +227,7 @@ def report_export
         end
       end
 
-      if report.modes == [:stacked_chart]
+      if report.modes == [Report::MODES[:stacked_chart]]
```

Comment:
On one hand we're not gaining anything in terms of runtime guarantees here, but on the other this provides a pointer to the source of possible values, so I think overall it's okay. 👍 

Would be cool if this would raise an exception on unknown value, but I realize our internal `Enum` class also just returns `nil` ...

Reply:
Yeah it's kind of a placebo, the main point is to have a source of truth :)